### ISSUE TITLE
Socket.IO and IISNode

### ISSUE THREAD

**Issue by Dunadan** [tjanczuk/iisnode#35]
Tomasz, how is it possible to include Socket.IO client library using IIS node?
According to official site recommendations, browser must follow url like http://[daemon]:[port]/socket.io/socket.io.js
For e.g., daemon runs as /server.js with rule 

add name="iisnode" path="/server.js" verb="*" modules="iisnode" 

I should use a rewrite of some sorts or there is another, better way?

**Comment by cesarvega** [comment]
this is not working on iis 7

**Comment by lagost** [comment]
I followed the steps to install iisnode on windows server 2012 with IIS 8.
The application loads but it doesn't seem to be reading the server.js file. It seems to load the html page and make no socket connections.
 
Files are all located under the root directory/node/.
I couldn't get the index.html file to find /socket.io/socket.io.js so I pointed to the file using the full file path.

The instructions for running iisnode in windows
[web.zip](https://github.com/tjanczuk/iisnode/files/2611504/web.zip) need more clarity.
